ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java

```diff
@@ -24,32 +24,36 @@ public BearerTokenAuthHandler(TokenValidator validator) {
 	@Override
 	public void handle(Context ctx) throws Exception {
 		String authHeader = ctx.getRequest().getHeaders().get(HttpHeaderNames.AUTHORIZATION);
-		if (authHeader != null && authHeader.startsWith("Bearer")) {
-			//Confirm token is valid
+		String token = ctx.getRequest().getQueryParams().get("authToken");
```

Comment:
You can pull the token off with a different handler that is very not standard and I don't want to put that in the main lib.